### ISSUE TITLE
Card carousel for @elementor pro users 

### ISSUE THREAD

**Issue by OzOseph** [elementor/elementor#13054]
Card carousel is a cornerstone this days in UX design, how come you don't offer it to pro users, and so many other plugins do offer that?

 @elementor please add Card carousel feature

**Comment by Jackevangelides** [comment]
Yes, I was also thinking something along these lines also.  It would be good if there was a widget that is a carousel section, and you can add/customise the section with whatever you want, but that section acts as a carousel, you can add slides as you wish and have all the carousel functionality, but you can add in widgets into the carousel section, allowing you to customise the content, as opposed to having to use only media or images etc...

**Comment by soundplay-production** [comment]
Would be great!

**Comment by 79ho3ein** [comment]
+1

**Comment by 79ho3ein** [comment]
@DorShahar  
Does elementor have a plan to add this feature? Are we waiting for it?

**Comment by IlPizza79** [comment]
+1, it's nice that Elementor always adds new and modern features... but please don't forget "basic" features like a simple post carousel :-(

**Comment by IlPizza79** [comment]
Another year has passed... and still nothing.
But wow, we finally have "Number Scrubber"!
